projects/distributed-tracing/src/shared/dashboard/dashboard-wrapper/navigable-dashboard.component.ts

```diff
@@ -100,6 +104,13 @@ export class NavigableDashboardComponent implements OnChanges {
     }
   }
 
+  public applyVariablesToDashboard(dashboard: Dashboard, variables: Dictionary<unknown> = {}): void {
+    for (const key of Object.keys(variables)) {
+      dashboard.setVariable(key, variables[key]);
+    }
+    dashboard.refresh();
```

Comment:
The refresh shouldn't be needed, but let's verify that this applies before the first fetch.